src/components/Footer.js

```
@@ -0,0 +1,243 @@
+    this.acknowledgeVersion = this.acknowledgeVersion.bind(this);
+
+    // eslint-disable-next-line no-undef
+    const commit = SECRETIN_APP_COMMIT;
+
+    const acknowledgedVersion = window.localStorage.getItem(
+      LOCAL_STORAGE_ACKNOWLEDGE_VERSION_KEY
+    );
+
+    this.state = {
+      commit,
+      acknowledgedVersion,
+      news: [],
+      showNews: acknowledgedVersion ? false : true,
+    };
+    console.log(this.state);
+  }
+
+  showNews() {
+    this.setState({ showNews: true });
+  }
+
+  hideNews() {
+    this.setState({ showNews: false });
+  }
+
+  acknowledgeVersion() {
+    window.localStorage.setItem(
+      LOCAL_STORAGE_ACKNOWLEDGE_VERSION_KEY,
+      this.state.commit
+    );
+    this.setState({
+      news: [],
+      acknowledgeVersion: this.state.commit,
+      showNews: false,
+    });
+  }
+
+  componentDidMount() {
+    if (this.state.acknowledgedVersion === null) {
+      return;
+    }
+
+    const somethingNew =
+      this.state.commit !== DEFAULT_DEVELOP_COMMITISH &&
+      this.state.acknowledgedVersion !== this.state.commit;
+    if (somethingNew) {
+      fetch(
+        'https://raw.githubusercontent.com/secretin/secretin-website/develop/changelog.json'
+      )
+        .then(response => {
+          return response.json();
+        })
+        .then(data => {
+          const news = [];
+          let foundAcknowledgedVersion = false;
+          for (const change of data.changelog) {
+            if (change.version === this.state.acknowledgedVersion) {
+              foundAcknowledgedVersion = true;
+              break;
+            }
+            news.push(change);
+          }
+
+          if (foundAcknowledgedVersion) {
+            this.setState({ news });
+          } else {
+            // If we didn't find the acknowledged version, then we didn't updated changelog properly
+            // we will just show the last change then
+            this.setState({ news: [data.changelog[0]] });
+          }
+        });
```

Add a catch here to avoid crashing the app if anything goes wrong (this feature is not vital to the app). Logging the error and swallowing it should be enough.